tests/unit/workflows/test_workflows_tasks_submission.py

```diff
@@ -22,22 +22,380 @@
 
 from __future__ import absolute_import, division, print_function
 
-from inspirehep.modules.workflows.tasks.submission import add_note_entry
+from inspirehep.modules.workflows.tasks.submission import (
+    submit_rt_ticket,
+    create_ticket,
+    reply_ticket,
+    close_ticket,
+    send_robotupload,
+    add_note_entry,
+    filter_keywords,
+    prepare_keywords,
+    user_pdf_get,
+    prepare_files,
+    remove_references,
+)
 
+from six import StringIO
+from mock import MagicMock
+import mock, pytest
 
-class StubObj(object):
-    def __init__(self, data, extra_data):
+
+class MockObj(object):
+    class MockLog(object):
+        def __init__(self):
+            self._log = StringIO()
+
+        def info(self, *args, **kwargs):
+            self._log.write(args[0])
+
+        def error(self, *args, **kwargs):
+            self.info(args[0])
+
+        def debug(self, message, format=None):
+            self.info(message)
```

Comment:
Uhm, these three logging methods have all different signatures and implementations, despite being (essentially) the same thing....

Also, with this design one cannot see if the proper logging level happened. Maybe we should have three `StringIO` objects for the three different log levels? Maybe we should add some kind of marker (like `DEBUG:`) in front?